lib/ViewModels/GazetteerSearchProviderViewModel.js

```diff
@@ -113,6 +114,24 @@ GazetteerSearchProviderViewModel.prototype.search = function(searchText) {
     });
 };
 
+// Given a list of results sorted in decreasing importance, strip results that are close to another result with the same name
+function stripDuplicates(results) {
+    var i;
+    var placeshash={};
+    var stripped = [];
+    for (i=0; i < results.length; i++) {
+        var lat = Number(results[i].location.split(',')[0]).toFixed(1);
```

Comment:
Yes. "Pretty far apart" meaning 0.1 degrees of latitude or longitude, or ~20km. This was intentional, so the user isn't presented with meaningless distinctions like "Sydney", "City of Sydney", etc.